botcity/core/bot.py

```diff
@@ -902,29 +920,33 @@ def copy_to_clipboard(self, text, wait=0):
         delay = max(0, wait or config.DEFAULT_SLEEP_AFTER_ACTION)
         self.sleep(delay)
 
-    def tab(self, wait=0):
+    def tab(self, wait=0, presses=1):
         """
         Press key Tab
 
         Args:
             wait (int, optional): Wait interval (ms) after task
+            presses (int, optional): Number of times to press the key. Defaults to 1.
 
         """
-        pyautogui.press('tab')
-        delay = max(0, wait or config.DEFAULT_SLEEP_AFTER_ACTION)
-        self.sleep(delay)
+        for i in range(presses):
+            self._kb_controller.tap(Key.tab)
+            self.sleep(config.DEFAULT_SLEEP_AFTER_ACTION)
```

Comment:
We need to add the logic to pick the DEFAULT or the wait, whichever is larger.
```
delay = max(0, wait or config.DEFAULT_SLEEP_AFTER_ACTION)
```

Since we are always executing the `self.sleep` after the action, the `self.sleep` outside the for loop must be removed.